### PR TITLE
Batch gets with radix-sort

### PR DESCRIPTION
**Pull request by SoujanyaPonnapalli**

### CODE REVIEW

File: src/index.spec.ts [hunk 1]
```diff
@@ -30,13 +30,23 @@ describe('Try original simple-save-retrieve', () => {
     const val = tree.get('test');
     should.exist(val.value);
     val.value!.should.equal('one');
+    const val1 = tree.batchGet([Buffer.from('test')]);
+    should.equal(val.value, val1[0].value);
```
Comment (no2chem) [inline src/index.spec.ts]: In the future, please use a deep equals... val.value.should.deep.equals(). Also this belonged in another test.

File: src/index.spec.ts [hunk 2]
```diff
@@ -64,6 +79,11 @@ describe('Try original simple-save-retrieve', () => {
     const val = tree.get('doge');
     should.exist(val.value);
     val.value!.should.equal('coin');
+    const val1 = tree.batchGet([Buffer.from('doge')]);
```
Comment (no2chem) [inline src/index.spec.ts]: Belongs in another test...

**Comment by no2chem** [comment]
@SoujanyaPonnapalli I merged this, but please read the comments. In the future, additional functionality belongs in new tests, not existing ones.

**Comment by SoujanyaPonnapalli** [comment]
Sure, shall I go ahead and add new tests and make the changes? It shouldn't take much time anyway.

**Comment by no2chem** [comment]
Can you refactor out those tests in the next hour or so? The way they're currently written using a non-deep compare is breaking some changes I'm trying to oush

**Comment by SoujanyaPonnapalli** [comment]
Yea sure! Will get it done.

**Pull request merged**